website_sale_attribute_filter_category/static/src/js/website_sale_attribute_filter_category.js

```diff
@@ -0,0 +1,31 @@
+odoo.define('website_sale_attribute_filter_category.animation', function (require) {
+'use strict';
+
+var core = require('web.core');
+var sAnimation = require('website.content.snippets.animation');
+
+
+sAnimation.registry.sale_attributes_filter_category = sAnimation.Class.extend({
```

Comment:
Why is this needed? Isn't there a class inside Bootstrap for collapsable content without JS?

Reply:
Is only for icon >

Reply:
Is this the way is done in Collapsable product categories?

Reply:
If needed, please put a comment about what this does

Reply:
https://github.com/odoo/odoo/blob/2156af480fddc09b5897b349c04c4c9a2d67c802/addons/website_sale/static/src/js/website_sale.js#L58

Reply:
I did not see these lines. Thanks
I think that with bootstrap4 is not necessary